ffmpeg/ffmpeg.go

```diff
@@ -68,6 +68,9 @@ type TranscodeOptions struct {
 type MediaInfo struct {
 	Frames int
 	Pixels int64
+	//for dnn result
+	Isdnn     bool
+	Dnnresult string
```

Comment:
What is the format of `Dnnresult`? Would be good to document.

If it is an empty string can we assume that the lvpdnn filter was not used? If so, could we remove `Isdnn` and just check if `len(Dnnresult) == 0` to determine if a DNN result was returned or not?